mojave/code/modules/client/preferences/fatness.dm

```diff
@@ -14,6 +14,11 @@
 /datum/preference/choiced/fatness/apply_to_human(mob/living/carbon/human/target, value)
 	var/lowertext_value = lowertext(value)
```

Comment:
This var can be removed, it's not used anymore since the code was commented out.

Reply:
the commented out code can be nuked, but the fatness var needs to stay since its used to actually apply fatness filters

Reply:
I removed the commented out part. Idk why but I just figured it wouldn't hurt to keep there in case we reworked the way stuff worked later on but... it's like 3 lines anways.

The actual rest of it is needed, as it'll break the entire thing without it, so that remains there.

Reply:
Sorry, I meant to say change `target.fatness = lowertext_value` to `target.fatness = lowertext(value)`.  The `lowertext_value` is just an intermediate variable that doesn't really need to exist.